packages/confluence-adapter/src/definitions/transformation_utils/space.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ *                      Copyright 2024 Salto Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { definitions } from '@salto-io/adapter-components'
+import _ from 'lodash'
+import { values as lowerdashValues } from '@salto-io/lowerdash'
+import { assertValue } from './generic'
+
+export const adjustHomepageToIdFunc: definitions.AdjustFunction = item => {
+  const value = assertValue(item.value)
+  const homepage = _.get(value, 'homepage')
+  if (!lowerdashValues.isPlainRecord(homepage) || !('id' in homepage)) {
```

Comment:
nit: please use 
```suggestion
  if (!lowerdashValues.isPlainRecord(homepage) || !Object.prototype.hasOwnProperty.call(homepage, 'id')) {
```
or something else that is guaranteed to look at the homepage and not some prototype (should probably be ok in this case, but we try not to have these in the code because they sometimes have unexpected results e.g. on `toString`)